src/main/java/DayTwo.java

```diff
@@ -2,21 +2,25 @@
 import java.util.List;
 
 public class DayTwo {
+    //can we remove this state?
     int totalFuel =0;
 
+    //choice of parameter type?
     public int getFuel(String fuels) {
 
         List<String> listOfFuels = Arrays.asList(fuels.split("\n"));
         for (String fuel:
                 listOfFuels) {
             int addedFuel =0;
+            //interesting typing as before plus duplication
```

Comment:
Again another case for `parseInt` and I'm not sure the `floor` is needed. We've also done the calculation twice, which seems to be unnecessary.